test/lib/index.spec.js

```diff
@@ -180,6 +184,245 @@ describe('The API', function() {
     });
   });
 
+  describe(`mock.bypass()`, () => {
+    afterEach(mock.restore);
+
+    it('bypasses mock FS', () => {
+      mock({'/path/to/file': 'content'});
+
+      assert.equal(fs.readFileSync('/path/to/file', 'utf8'), 'content');
+      assert.throws(() => fs.readFileSync(__filename));
+      assert.doesNotThrow(() => mock.bypass(() => fs.readFileSync(__filename)));
```

Comment:
Instead of asserting that `mock.bypass` doesn't throw, I think it would be a more useful test to assert that the contents of a file read within a `mock.bypass` function are what we expect.  Or even just to assert that the return from `fs.access` (or `fs.exists`) is what we expect.

Maybe something like this
```js
let exists = false;
mock.bypass(() => {exists = fs.accessSync(__filename)});
assert.ok(exists);
```

Reply:
`accessSync` throws if it can't be accessed, also. Agreed on existsSync being more useful. I'll update it.